services/Makefile

```diff
@@ -9,6 +9,24 @@ endif
 
 all: check docs container
 
+# Checking to see if the directory exists because npm install updates the
+# directory every time it runs, busting the GNU/Make cache causing rebuilds
+depends: package.json
+	@if [ ! -d node_modules ]; then \
+		npm install; \
+	fi;
+
+docs: depends
+
+store-commit:
```

Comment:
Shouldn't this now go into the `node.mk` file?

Reply:
`distribution/Makefile` doesn't include `node.mk`